frontend/app/views/fragments/promos/promoPrimary.scala.html

```diff
@@ -14,7 +14,7 @@ <h2 class="promo-primary__title h-promo@if(isLead){ h-promo--primary}">@title</h
     </div>
     <div class="promo-primary__media@if(stampImageOpt){ stamped-image}">
         <div class="promo-primary__image">
-            <img src="@Asset.at(imageUrl)" alt="@title" class="responsive-img">
+            <div class="js-imager-loaded-image" data-src="@imageBasePath/{width}.jpg" data-available-widths="1000,500,140" data-alt="@title" data-class="responsive-img"></div>
         </div>
```

Comment:
@feedmypixel I think I'm happy to leave this as it at the moment. My plan is to try a picture/srcset polyfill in the near future so will likely revisit that then.